src/libPMacc/include/math/vector/Vector.hpp

```diff
@@ -554,7 +556,7 @@ template <typename, int> class T_Storage
 >
 HDINLINE Vector<T_Type, T_Dim>
 operator+(const Vector<T_Type, T_Dim, T_Accessor, T_Navigator, T_Storage>& lhs,
-          const T_Type& rhs)
+          typename boost::call_traits<T_Type>::param_type rhs)
```

Comment:
please use `typename boost::call_traits<typename Vector<T_Type, T_Dim, T_Accessor, T_Navigator, T_Storage>::type>>::param_type rhs`
This is also wrong in the default implementation